browser_history/cli.py

```diff
@@ -23,11 +25,18 @@ def make_parser():
                                             Checkout the GitHub repo https://github.com/pesos/browser-history
                                             if you have any issues or want to help contribute''')
 
+    parser_.add_argument('-t','--type',
+                        default='all',
+                        help=f'''
+                                argument to decide whether to retrieve history or bookmarks.
+                                Should be one of all, {AVAILABLE_TYPES}.
+                                Default is all (gets both histories and bookmarks)
+                                .''')
     parser_.add_argument('-b', '--browser',
                          default='all',
                          help=f'''
-                                browser to retrieve history from. Should be one of all, {AVAILABLE_BROWSERS}.
-                                Default is all (gets history from all browsers).''')
+                                browser to retrieve history or bookmarks from. Should be one of all, {AVAILABLE_BROWSERS}.
+                                Default is all (gets history or bookmarks from all browsers).''')
```

Comment:
IMO we should keep the default to history since that is the primary purpose of this package.

Reply:
Okay